packages/components/src/List/ListComposition/Manager/hooks/useLocalStorage.hook.js

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react';
+
+export default function useLocalStorage(localStorageKey, initialValue) {
+	const [value, setValue] = useState(localStorage[localStorageKey] || initialValue);
```

Comment:
localStorage is signifcantly slower than in memory and also DOM blocking
what about having an intermediate in memory object and async access to localstorage ?

Reply:
You need to temper that. It's not a problem, there is no perf issue at all. This is done at user action, so this is not very costly.